USAGE.rst

```diff
@@ -29,6 +30,16 @@ As in our :ref:`compiling from source <install-source>` section, we need a few d
     # PIConGPU simulation output
     mkdir $SCRATCH/runs
 
+Download the PIConGPU source code either from the main repository or from your fork.
```

Comment:
Please remove, this is *not* enough to get PIConGPU source, tools, environment, etc. set up. See the above linked and previously mentioned dependencies and picongpu.profile sections.